pkgs/by-name/ca/cargo-autoinherit/package.nix

```
@@ -0,0 +1,25 @@
+{ lib, rustPlatform, fetchFromGitHub }:
+
+rustPlatform.buildRustPackage rec {
+  pname = "cargo-autoinherit";
+  version = "0.1.4";
+
+  src = fetchFromGitHub {
+    owner = "mainmatter";
+    repo = pname;
+    rev = "v${version}";
+    hash = "sha256-BuZDCd4SwSYg5eKV61L3RpPVmq5NZDAOc9zOz5QiSNI=";
+  };
+
+  cargoHash = "sha256-9hhrVkC1xB2E/vatkiM4PIJyXq+0GDoqlgXZXc8WehU=";
+
+  meta = with lib; {
+    description = "(Auto)DRY for your Rust dependencies";
+    homepage = "https://github.com/mainmatter/cargo-autoinherit";
+    license = with licenses; [ asl20 /* OR */ mit ];
+    platforms = platforms.linux; # not tested on others
```

```suggestion
    platforms = platforms.unix;
```

Would be fine. If you find it failing to build on CI (ofBorg/Hydra), just mark it as broken